packages/matchbox/src/components/Connect/Connect.tsx

```diff
@@ -3,10 +3,13 @@ import { Box } from '../Box';
 import { tokens } from '@sparkpost/design-tokens';
 import styled from 'styled-components';
 
-const FocusHandler = styled(Box)`
+// $zIncrement is here to ensure the middle element appears
+// above side elements until it side elements are focused
+const FocusHandler = styled(Box)<{ $zIncrement: number }>`
+  z-index: ${(props) => tokens.zIndex_default + props.$zIncrement};
   &:focus-within {
     position: relative;
-    z-index: ${tokens.zIndex_default};
+    z-index: ${tokens.zIndex_default + 2};
```

Comment:
This works but seems a little odd, maybe we need to consider a new zIndex token?

Reply:
would `above` work? with a value of 10 just to be safe?

Reply:
Yea let's do that